commands/stats.go

```diff
@@ -0,0 +1,21 @@
+package stats
+
+
+// HandleStats handles a stat command
+func HandleStats(i *dgo.InteractionCreate, s *dgo.Session) {
+// Gathers stats
+	// Gathers number of bots
+		numberOfBots := "holderoftheplaces"
+	// Gathers number of members
+		numberOfPariticipants := "holderoftheplaces"
+	// Gathers number of members online
+		numberOfPariticipantsOnline := "holderoftheplaces"
+	// Gathers numbers of people with staff role
+		numberOfStaff := "holderoftheplaces"
+	// Gathers numbers of people with staff role that are online
+		numberOfStaffOnline := "holderoftheplaces"
+	// Sends stats
+	utils.SendResponse("There are " numberOfBots " many bots", i, s)
```

Comment:
There should be + operators between strings that you are combining 
**"There are " +  numberOfBots + " many bots"**

Reply:
also you then also have to cast the value to a string to add it to another string
 
**string(numberOfBots)**